src/Symfony/Component/HttpClient/HttpClientTrait.php

```diff
@@ -116,6 +116,15 @@ private static function prepareRequest(?string $method, ?string $url, array $opt
             $options['peer_fingerprint'] = self::normalizePeerFingerprint($options['peer_fingerprint']);
         }
 
+        if (isset($options['crypto_method']) && !\in_array($options['crypto_method'], [
+            \STREAM_CRYPTO_METHOD_TLSv1_0_CLIENT,
```

Comment:
Shouldn't we define our own constants here, rather than leaking constants from the PHP streams API?